### ISSUE TITLE
Asterisk within backticks identified as for emphasis

### ISSUE THREAD

**Issue by treyhunner** [rvben/rumdl#49]
This file:

```md
The `__mul__` method is needed for left-hand multiplication (`vector * 3`) and `__rmul__` is needed for right-hand multiplication (`3 * vector`).
```

Shows linting errors due to the `*` inside the backticks:

```bash
$ uv tool run rumdl check --disable MD013,MD041 file18.md
file18.md:1:70: [MD037] Spaces inside emphasis markers: "* 3`) and `__rmul__` is needed for right-hand multiplication (`3 *" [*]

Issues: Found 1 issues in 1 file (30ms)
Run with `--fix` to automatically fix 1 of the 1 issues
```

**Comment by rvben** [comment]
Thanks for the issue @treyhunner . I've made fixes to make sure inline code is not giving false positives like this. Also took the opportunity to look at other rules for similar issues. These should be fixed in 0.0.117.
